.github/workflows/dockerimage.frontend.yml

```diff
@@ -0,0 +1,57 @@
+on:
+    push:
+      branches:
+        - develop
+        - master
+      paths:
+        - "frontend/src/**"
+    pull_request:
+      branches:
+        - develop
+        - master
+      paths:
+        - "frontend/src/**"
+
+name: CI-FrontEnd
+
+env:
+    IMAGE_NAME: ${{ secrets.DOCKER_HUB_ORGANIZATION }}/stamacasa
+
+jobs:
+    build-parser:
+      name: Build FrontEnd container
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkoout
+          uses: actions/checkout@v2
+
+        - name: Build Docker image
+          run: |
+            cd frontend/src
+            docker build . \
+              --tag $IMAGE_NAME:$GITHUB_SHA \
+              --file Dockerfile
```

Comment:
Wouldn't this expect the `Dockerfile` to be in `frontend/src`, whereas we have it now in `frontend` directly